src/modules/timeTracker/timeTracker.sagas.js

```diff
@@ -0,0 +1,31 @@
+import { takeLatest, put } from 'redux-saga/effects';
+
+import { getFirestore } from '../../shared/services/firestore';
+import reportError from '../../shared/utils/reportError';
+import { TimeTrackerTypes, TimeTrackerActions } from './timeTracker.redux';
+
+export function* saveResult() {
+  try {
+    yield put(TimeTrackerActions.stop());
+    const db = yield getFirestore();
+    yield db
+      .doc('benchmark-1')
```

Comment:
To nie miało wyglądać tak?
-- pwa
---- benchmark-1
------ { ...kolekcjaResultatow }